src/asm_syntax.hpp

```diff
@@ -14,17 +14,22 @@ using label_t = std::string;
 }
 using crab::label_t;
 
+// Abstract Syntax Machine (ASM) syntax.
```

Comment:
I used `asm` as shorthand for "assembly" :) (as opposed to crab-related parts). If it's not obvious, then it's worth renaming too. Probably just `abstract_syntax` for this file.

The namespace is here mostly for doxygen to cluster the related classes together; that's why `using` follows immediately.

Reply:
Updated comment